ui/pages/keychains/reveal-seed.js

```diff
@@ -144,9 +144,39 @@ const RevealSeedPage = () => {
   };
 
   const renderRevealSeedContent = () => {
+    // default fir SRP_VIEW_SRP_TEXT event because this is the first thing showed after rendering
```

Comment:
Tiny verbiage nits:

* I think `fir` is meant to be `for`
* `showed` can be `shown`